.github/workflows/main.yml

```diff
@@ -30,3 +30,9 @@ jobs:
 
       - name: test
         run: npx tree-sitter test
+
+  tests:
+    uses: ./.github/workflows/nvim-tests.yml
+    with:
+      neovim_versions: |
+        [ "nightly", "v0.10.2", "v0.10.1", "v0.9.5" ]
```

Comment:
nvim-treesitter no longer supports v0.9 and earlier.

Reply:
thanks!